docs/ConfigurationFile.md

```diff
@@ -323,7 +323,8 @@ exports.config = {
     afterSession: function (config, capabilities, specs) {
     },
     /**
-     * Gets executed after all workers got shut down and the process is about to exit.
+     * Gets executed after all workers got shut down and the process is about to exit. An error
+     * thrown in the onComplete hook will result in the test run failing.
```

Comment:
Can we make this update everywhere?

Reply:
I'm not sure where else you want this updated, I can't find any other description of onComplete to update in any .md file.

Reply:
e.g. /packages/wdio-cli/src/templates/wdio.conf.tpl.ejs .. there are a couple of wdio.conf.js examples in the repo

Reply:
Ah okay, I found the culprits and updated them.